examples/llm_inference/android/app/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.example.mediapipe.llminference"
+    compileSdk = 34
```

Comment:
Can we use 32 like here: https://github.com/googlesamples/mediapipe/blob/main/examples/hand_landmarker/android/app/build.gradle

Reply:
Android developers are encouraged to always compile against the latest SDK, it ensures that the app works in the latest versions of the Android OS.